src/robusta/utils/base64_utils.py

```diff
@@ -0,0 +1,14 @@
+import base64
+import logging
+
+
+def is_base64_encoded(value: str) -> bool:
+    try:
+        if base64.b64encode(base64.b64decode(value)) == bytes(value, 'utf-8'):  # key is b64
```

Comment:
I think you can just do:
`return base64.b64encode(base64.b64decode(value)) == bytes(value, 'utf-8')`  # value is b64

Reply:
Understood, fixed.